src/test/java/tests/EntityDetailTabTests.java

```diff
@@ -0,0 +1,39 @@
+package tests;
+
+import beans.menus.EntityDetailListResponse;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import utils.TestContext;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = TestContext.class)
+public class EntityDetailTabTests extends BaseTestClass {
+
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+        configureRestoreFactory("rmnchdomain", "rmnchusername");
+    }
+
+    @Override
+    protected String getMockRestoreFileName() {
+        return "restores/rmnch.xml";
+    }
+
+    // Regression test confirming that variables are passed to detail tab sub contexts
+    @Test
+    public void testDetailTabChildVariable() throws Exception {
+        // just need to ensure this doesn't error
```

Comment:
still don't think these types of tests are great. if this ever fails it becomes very difficult to figure out what exactly this was testing since the restore / app is so large